### DISCUSSION TITLE
Save state in toggle quick terminal?

### DISCUSSION THREAD

**Discussion by krlsdgzmn** [ghostty-org/ghostty#3649] [Q&A] [answered]
is there a way to save the state of quick terminal after closing ghostty?
<img width="1440" alt="Screenshot 2024-12-28 at 12 54 38 PM" src="https://github.com/user-attachments/assets/55cc43d3-d36d-4b95-9034-6914b33ab1dc" />

**Comment by 00-kat** [comment]
Ghostty does not currently support sessions, so I'm fairly certain that this isn't possible right now either.

**Comment by mitchellh** [comment, marked as answer]
#8399
